pages/blog/2020-summary.md

```diff
@@ -308,9 +308,9 @@ What is the benefit of writing/republishing on the AsyncAPI blog? Visibility amo
   </tr>
   <tr>
     <td align="center">
-      <a href="/blog/automated-releases">
+      <Link href="/blog/automated-releases"><a>
       Full automation of release to NPM and Docker Hub with GitHub Actions and Conventional Commits
-      </a> (March 2020)
+      </a></Link> (March 2020)
```

Comment:
Please format that 😄  and of course, please do this also in another places in that markdown :)

Reply:
Also please add `cursor: pointer` behaviour because I don't see any pointer when I hover that links in blog posts. We should also check that behaviour in other places, not only in blog posts, but also in the source code, in the places where you added `<Link/>` component.

Reply:
@magicmatatjahu Fine, I will see all the markdown files to add `Link` component and add `cursor:pointer` everywhere required. 

Will you please tell me which formatter is used to format the code in this repository? I had used `Prettier` before in this PR but got so many changes in the file that I reverted it back. So, I am a bit confused about what to use.

Reply:
> Will you please tell me which formatter is used to format the code in this repository? I had used Prettier before in this PR but got so many changes in the file that I reverted it back. So, I am a bit confused about what to use.

Atm we don't use any formatter, but we should use `eslint`, but it's a task for another PR.

Reply:
md files won't be changed by eslint at all. I have a suggestion for the PR targeting format the code, that we can make an `npm script` in the `package.json` file, for formatting the `js` files using eslint/Prettier. Using that script, it will format all the files in a single commit. For now, I will only format wherever I think it will be needed in `js` files.

Reply:
> md files won't be changed by eslint at all

`eslint` as far as I know has the ability to parse files other than JS, but you need to configure it. We should see if it is possible.

> I have a suggestion for the PR targeting format the code, that we can make an npm script in the package.json file, for formatting the js files using eslint/Prettier. Using that script, it will format all the files in a single commit.

We have similar solution in the other repos using `husky`. If you wanna handle that, please first open issue for that and start implementation :)

Reply:
>We have similar solution in the other repos using husky. If you wanna handle that, please first open issue for that and start implementation :)

I saw this on `asyncapi/react` repository, can I make something like this? It's easy and I have implemented it before as well in my own repositories. Since we are not working with typescript, I can use `eslint` with this.

![image](https://user-images.githubusercontent.com/76521428/152799029-0cde76fb-1436-42a6-8705-c7503d593125.png)

Reply:
Yeah, you can, but better will be base on that config https://github.com/asyncapi/markdown-template/blob/master/.eslintrc :)